ontologies/Journal_with_items.md

```diff
@@ -0,0 +1,35 @@
+
+#JSON examples journals
+
+Comment line 13-22: 
+a) are we happy dealing with all these different sorts of title as subsets of alternativeTitlt?
+b) if so, keyTitle and uniformTitle need to be added to the ontology and described
+
+Comment line 43: "designation" needs to be added the ontology
+It corresponds in part to the start date and end date properties that are already there, covering dates of publication and sequential designation, but has other things mixed in like volumes
+
+Comment line 44 onwards: ISSN is not in the ontology yet
+It occupies the same field as ISBN and the distinguishing feature is apparently the format: ISSN has four digits, a slash then another four digits.
+Transformer would need to be programmed to understand this distinction
+
+Comment line 57: Format in this instance just duplicates the content of WorkType - surely we don't need both
+
+Comment line 58: frequency can be either current or past: the distinction lies in which MARC field it comes from (310 or 321 respectively)
+
+Comment line 88: numbering is not yet in ontology
+
+Comment line 89: supplement is not yet in ontology
+
+Comment lines 90-108: relatedMaterialNote brings together material fromn MARC 530 - the alternative formats field - and MARC 775, related to.  We would want to distinguish these?
```

Comment:
Should that be 776 (Additional physical form entry) rather than 775 (other edition entry)?